src/amo/components/SearchPage.js

```diff
@@ -27,20 +22,19 @@ export default class SearchPage extends React.Component {
 
   static defaultProps = {
     LinkComponent: Link,
-    ResultComponent: SearchResult,
     filters: {},
     pathname: '/search/',
     results: [],
   }
 
   render() {
     const {
-      LinkComponent, ResultComponent, count, filters, hasSearchParams,
-      loading, page, pathname, results,
+      LinkComponent, count, filters, hasSearchParams, loading, page, pathname,
+      results,
     } = this.props;
     const queryParams = this.props.queryParams ||
       convertFiltersToQueryParams(filters);
-    const paginator = count && hasSearchParams > 0 ? (
+    const paginator = count > 0 && hasSearchParams ? (
```

Comment:
since `count` is always used like a number, maybe make the property default to 0 for sanity?